core/src/main/java/com/novoda/landingstrip/TabsContainerView.java

```diff
@@ -0,0 +1,20 @@
+package com.novoda.landingstrip;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.LinearLayout;
+
+public class TabsContainerView extends LinearLayout {
+
+    public TabsContainerView(Context context) {
+        super(context);
+        super.setOrientation(HORIZONTAL);
+    }
+
+    public void setActivated(int position) {
+        for (int index = 0; index < getChildCount(); index++) {
```

Comment:
Would it be worth having a reference to the last activated view instead of iterating over all the children? Or would that be error-prone?